src/ast/passes/semantic_analyser.cpp

```diff
@@ -1655,6 +1655,8 @@ If you're seeing errors, try clamping the string sizes. For example:
       }
     }
     call.return_type = CreateUInt64();
+  } else if (call.func == "getopt") {
+    LOG(BUG) << "getopt function call should have been replaced at this point.";
```

Comment:
I don't think you need a dedicated `BUG` here as this should fail when it hits the final `else`. We should be okay with slowly eliminating function cases from the semantic analyzer, and not replacing everything with a call to `BUG` :)